.github/workflows/actions.yml

```diff
@@ -8,18 +8,15 @@ on:
     branches: '**'
 
 jobs:
-  build:
-    name: Build & Deploy
+  build-linux:
+    name: Build Linux & Deploy
```

Comment:
Sounds a bit misleading to me like we would only build the linux executable here